src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,42 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+export const Navigation = () => {
+  const [selectedNavLink, setSelectedNavLink] = useState<string>('');
+
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <Link
```

Comment:
use NavLink as misha does in his [video](https://mate.academy/learn/react/react-router#/video/3346)